### ISSUE TITLE
Non-english api return messages

### ISSUE THREAD

**Issue by SonGokussj4** [grokability/snipe-it#10587]
### Debug mode

- [x] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

API calls should return English messages all the time.

When I deleted departments from API, 

- upon success, the **Czech language** message is returned, 
- upon error **English one**...

My user (Admin) has Czech as his profile language. But that is not the problem.

### Reproduction steps

I called API for deleting departments two times

```bash
curl --request DELETE --url https://snipeit.ourcompany.cz/api/v1/departments/7 \
     --header "Authorization: Bearer $SNIPEIT_API" \
     --header 'Content-Type: application/json' \
     --header 'Accept: application/json'
```

Department is deleted just fine after the first call.

> Message after the first call (Czech):
```
{"status":"success","messages":"Odd\u011blen\u00ed bylo \u00fasp\u011b\u0161n\u011b smaz\u00e1no.","payload":null}
```

> Message after the second call (English):
```
{"status":"error","messages":"Department not found","payload":null}
```

### Expected behavior

I would expect english messages when returning anything from API. Most importantly not a bilingual response. 

```json
{
    "status":"success",
    "messages":"Department was successfully deleted.",
    "payload":null
}
```

### Snipe-IT Version

5.3.8

### Operating System

Docker

### Web Server

Docker - NGinx

### PHP Version

Docker

**Comment by snipe** [comment]
We don’t promise to have all translations completed. I don’t speak Czech, so I couldn’t translate the whole system into Czech if I wanted to. If you’d like to help out with translations, please see https://snipe-it.readme.io/docs/translations

<blockquote><img src="/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/translations">Translations</a></strong></div><div>Traduci! 翻訳！Oversette! Traduire! 번역하기! แปล! ¡Traducir! Przetłumacz! Übersetzen!</div></blockquote>

**Comment by SonGokussj4** [comment]
Sorry but that is not the problem I mentioned. I don't have a problem it's not all translated! I would gladly help with that.

I've got problem that when it returns `200 response`, it returns the message in **Czech** and if `404 response`, it returns the message in **English**. 

I think it should always return in **English**. 

> Or, at least, have a config in admin/profile to choose a language it would return in.
> I want to have the site in Czech but for API, it should be English. The advantage is there -> everything **is already in English**

**Comment by snipe** [comment]
If the API user (I assume that's your Czech admin user) is being used, it should always respond in Czech. The expected behavior here is that it should adopt the language of whatever user's API key is being used. (I assume that first message in Czech is something like "Department not found"?)

**Comment by SonGokussj4** [comment]
Yes. It is. Okay, I understand your point of view now. The failed API call should have returned:

```json
{"status":"chyba","messages":"Oddělení nenalezeno","payload":null}  <-- czech
// instead of
{"status":"error","messages":"Department not found","payload":null}  <-- english
```

So my modified propose would be to add an option/checkbox for the case the admin user wants to have Czech but return API responses in English? Is this viable?

Workaround for this would be creating new user, for example "apiadmin" with "English" as default language and do API calls through him.

**Comment by SonGokussj4** [comment]
Ok, small update. I've changed the admin user to English,US, restarted docker and it still returns czech response when I delete a department.

- Settings -> Localization Settings -> Default Language -> English, US
- Edit your profile -> Language -> English, US

I even tried to make a new Token. Still czech response.

**Comment by snipe** [comment]
I think ticket is absurdly old, but I think this would be resolved by switching the .env language.

**Comment by SonGokussj4** [comment]
Thanks for the response. I don't think I can test that, because in those 3 years I've stopped working in the company where I was implementing that. But thanks, I'll try to remember that if I ever encounter that again. :-)
